src/renderer.ts

```diff
@@ -819,21 +819,9 @@ const renderEditions = (format: Format, excludeStyles = false) => (
 			return h(EditionsPullquote, { quote, attribution, format, key });
 		}
 
-		case ElementKind.RichLink: {
-			const { url, linkText } = element;
-			return h(RichLink, { url, linkText, key, format });
-		}
-
 		case ElementKind.LiveEvent:
 			return h(LiveEventLink, { ...element, key });
 
-		case ElementKind.Interactive:
-			return h(Interactive, {
-				url: element.url,
-				key,
-				title: element.alt,
-			});
-
```

Comment:
I'm surprised TypeScript doesn't complaing that we didn't exhaust all options .. might be worth adding a default case returning `null` ?

Reply:
Good shout, updated!